frontend/src/components/LineChart/LineChart.js

```diff
@@ -0,0 +1,53 @@
+import { PropTypes } from 'prop-types'
+import { useMemo, useRef } from 'react'
+import { useTheme } from '@mui/system'
+import { VictoryAxis, VictoryChart, VictoryLine, VictoryScatter, VictoryTooltip } from 'victory'
+import Box from '@mui/material/Box'
+
+import { nFormatter } from 'utils/helpers'
+import { getLineChartDomain, useVictoryTheme, useClientSize } from 'utils/victory'
+import ChartFlyOut from 'components/ChartFlyOut'
+
+const LINE_CHART_RATIO = 0.75
+
+const LineChart = ({ data, title, xAxisLabel, yAxisLabel }) => {
+  const theme = useTheme()
+  const domain = useMemo(() => getLineChartDomain(data, LINE_CHART_RATIO), [data])
+  const victoryTheme = useVictoryTheme(theme)
+
+  const ref = useRef()
+  const size = useClientSize(ref)
+
+  return (
+    <Box ref={ref} sx={{ columns: 1, marginTop: '-50px' }}>
+      <VictoryChart {...size} theme={victoryTheme}>
+        <VictoryAxis />
+        <VictoryAxis dependentAxis domain={domain} tickFormat={t => nFormatter(t)} />
+        <VictoryLine data={data} />
+        <VictoryScatter
+          data={data}
+          labels={() => ''}
+          labelComponent={
+            <VictoryTooltip
+              flyoutComponent={<ChartFlyOut title={title} xAxisLabel={xAxisLabel} yAxisLabel={yAxisLabel} />}
```

Comment:
These tooltips are getting clipped, possibly by the column styling in SmallBusinessSupport. <img width="302" alt="Screen Shot 2022-02-28 at 6 58 28 PM" src="https://user-images.githubusercontent.com/244952/156090120-ce71af01-2a57-4b8b-bbfa-c5e8d2310d64.png"><img width="315" alt="Screen Shot 2022-02-28 at 6 58 55 PM" src="https://user-images.githubusercontent.com/244952/156090121-bee27329-afda-416d-acaf-f806d6b41d56.png">